specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -4384,6 +4511,23 @@
       ],
       "description": "Deployment What-if operation parameters."
     },
+    "ScopedDeploymentWhatIf": {
```

Comment:
>ScopedDeploymentWhatIf [](start = 5, length = 22)

This would also apply to the subscription scope /whatIf, right? If you do make that change, it will be flagged as a breaking change because linters will see an existing property becoming required. I would add a note that it was always required on the service side.

Reply:
Yeah it also applies to subscription scope /whatif. I can make the change, which will fail the breaking change checker for sure. I remember Tian had to revert the change for 2019-07-01 for subscription scope /validate. I guess we'll have to ask for a bypass?

Reply:
Up to you. I'm fine if we split that into separate PRs.

Reply:
On second thought I think we should send a separate PR, since the existing sub-level deployment /validate and PUT APIs will also need to be updated. Besides, I'll need to do more investigation to make sure it's safe to update them, because it is unclear to me that if it will break the generated SDKs (I have a bad feeling about Python).